src/index.ts

```diff
@@ -14,6 +15,7 @@ app.openAPIRegistry.registerComponent("securitySchemes", "Bearer", {
 });
 
 app.route("/auth", authRouter);
+app.route("/evaluation", evaluationRouter);
```

Comment:
evaluation route does not get defined here, it gets defined at the very end after middleware has been declared